libs/features/personalization/entitlements.js

```diff
@@ -0,0 +1,17 @@
+export const ENTITLEMENT_MAP = {
+  '9565ef55-faad-430b-b661-596ba7a036c4': 'all-apps',
+  'c8d50cc2-491e-48df-a1b0-1509f0ca7323': 'photoshop',
+};
```

Comment:
Should we leave some info here who's setting up those segments - or a wiki link?

Maybe more of a future discussion, but should the segment ID mappings be authorable? I can imagine there to be a lot of experiments/tests if a user is in segment X, button shows Y

Reply:
We considered using an excel sheet to map this, but decided against it due to performance implications when personalizing a page.  We need this info asap as it holds up LCP until we know what personalization needs to be done.

Reply:
Still code based but I added support for consumers to defined entitlements via `config.entitlements`